bom/pom.xml

```diff
@@ -27,7 +27,7 @@
         <infinispanVersion>8.0.2.Final</infinispanVersion>
         <mongodbVersion>3.0.3</mongodbVersion>
         <fongodbVersion>2.0.2</fongodbVersion>
-        <orientdbVersion>2.1.9</orientdbVersion>
+        <orientdbVersion>[2.1.9,)</orientdbVersion>
```

Comment:
We prefer to specify the exact version for dependencies, it makes everything more predictable